NitroxClient/Communication/Packets/Processors/RocketPreflightCompleteProcessor.cs

```diff
@@ -1,12 +1,15 @@
-﻿using NitroxClient.Communication.Packets.Processors.Abstract;
+﻿using NitroxClient.Communication.Abstract;
+using NitroxClient.Communication.Packets.Processors.Abstract;
 using NitroxClient.MonoBehaviours;
+using NitroxModel.Core;
 using NitroxModel_Subnautica.Packets;
 using UnityEngine;
 
 namespace NitroxClient.Communication.Packets.Processors
 {
     public class RocketPreflightCompleteProcessor : ClientPacketProcessor<RocketPreflightComplete>
     {
+        private IPacketSender packetSender;
```

Comment:
With the construtor:
```suggestion
        private readonly IPacketSender packetSender;
        
```